src/apify/_charging.py

```diff
@@ -328,6 +336,38 @@ def get_charged_event_count(self, event_name: str) -> int:
     def get_max_total_charge_usd(self) -> Decimal:
         return self._max_total_charge_usd
 
+    async def _fetch_pricing_info(self) -> _FetchedPricingInfoDict:
+        """Fetch pricing information from environment variables or API."""
+        # Check if pricing info is available via environment variables
+        if self._configuration.actor_pricing_info and self._configuration.charged_event_counts:
```

Comment:
```suggestion
        if self._configuration.actor_pricing_info is not None and self._configuration.charged_event_counts is not None:
```

Better to be more defensive here